W3ChampionsStatisticService/PersonalSettings/PersonalSettingsController.cs

```diff
@@ -104,5 +104,13 @@ public async Task<IActionResult> MigrateCountry()
 
             return Ok();
         }
+
+        [HttpPost("{battleTag}/upvote")]
+        public async Task<int> UpvotePlayer(string battleTag)
+        {
+            var result = await _personalSettingsRepository.Upvote(battleTag);
```

Comment:
like mentioned, multiple upvotes should not be possible

Reply:
It should be possible. The flow is as follows:

1. User clicks on upvote button in Twitch Ext
2. They receive a confirmation from twitch that they enough bits / purchase bits
3. If the purchase is completed we send a request to our backend to upvote

Reply:
But if I am smart, I just use this endpoint independantly from twitch and give me a million upvotes? Can we add a secret that is needed for the twitch extension? Or are the extension secrets open like on a website? Or maybe do this via cors, so only twitch can acces that endpoint. Dont know. But we would need some sort of validation within that handler.

Reply:
That's exactly the idea. Each request from Twitch extension will have a signed JWT. On the backend I will store Twitch secret to verify this JWT. Once verified, I can proceed with upvote

Reply:
Ok perfect, but then we wait until the jwt stuff is also included? I dont think people will really go through the code and search for this endpoint, but as it is connected to money, we should wait until the endpoint is secured. Or featuretoggle it.

Reply:
Yes, no need to merge now. I'll mark this as a WIP